src/components/SortPriceInput/SortPriceInput.tsx

```diff
@@ -2,30 +2,42 @@ import Select from "@mui/base/Select";
 import { Option } from "./Option";
 import { SelectOption } from "@mui/base";
 
-const renderValue = (option: SelectOption<string> | null) =>{
+const renderValue = (option: SelectOption<string> | null) => {
   if (option == null) {
     return <span>Sort by</span>;
   }
 
   return (
     <>
-    Price:
-    <span className="text-azure">
-      {option.label}
-    </span>
+      Price:
+      <span className="text-azure">{option.label}</span>
     </>
   );
-}
+};
+
+type Props = {
+  value: string;
+  optionList: { value: string; label: string }[];
+  onChange?: () => void;
+  onFocus?: () => void;
+  onBlur?: () => void;
```

Comment:
It would be better if we could access the value of the event

Check if you can replace `VoidFunction` with `React.HTMLAttributes<HTMLSelectElement>['onChange | onFocus | onBlur']` or just add the adapter to the component and transform the library event arguments